charts/m4d/values.yaml

```diff
@@ -166,6 +166,16 @@ katalogConnector:
 
   affinity: {}
 
+razee: {}
+#  Razee local deployment requires the url, user and password parameters
+#  Razee deployment with oauth API key authentication requires the apiKey parameter
+#  Razee deployment with IBM Cloud Satellite Config requires the iamKey parameter
+#  url: ""
+#  user: ""
+#  password: ""
+#  apiKey: ""
+#  iamKey: ""
+
```

Comment:
I don't think that this format will play nice with helm doc generators, but we can deal with that later. What's apiKey? I didn't see it used in our codebase

Reply:
`apiKey` will be added as soon as the upstream PR to the library goes through.